src/main/resources/feedbackQuestionConstSumEditFormTemplate.html

```diff
@@ -34,6 +34,11 @@
                                 name="${Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTSPEROPTION}"
                                 value="false" checked>
                     </div>
+                    <div data-toggle="tooltip" data-placement="top"
+                                title="Respondents will have to distribute the total points specified here among the 
+                                ${optionRecipientDisplayName}. e.g. If you specify 100 points here and if there are 3 
+                                ${optionRecipientDisplayName}, respondents will have to distribute 100 points among 3 
+                                ${optionRecipientDisplayName}.">
```

Comment:
Indentation should be 8 spaces from line above (actually might be 4 spaces, or aligned with first word after the space, but it seems like this file is using the 8 spaces indentation).